### DISCUSSION TITLE
Companion App collection every minute?

### DISCUSSION THREAD

**Discussion by buxxdev** [NightscoutFoundation/xDrip#2935] [Q&A]
Hi, we’re trialling Freestyle Libre 3 with CamAPS thats providing BG readings to the Lock Screen notification every minute.
Is there any possibility that xdrip could read these values every minute instead of every 5 mins?
Thanks.

**Comment by Navid200** [comment]
The move to 1 reading every minute is not going to be a rapid move.
This is an example of a step:  https://github.com/NightscoutFoundation/xDrip/issues/2680

There is a lot to test.
The companion app mode itself is something relatively new.
